README.md

```diff
@@ -440,7 +440,7 @@ type BlogPost =
   }
 ```
 
-Our API tells us the author of the blog post as a string and whether we follow them as a boolean. This admits more cases than are actually possible -- you can't follow yourself, for example -- so we are more precise and model an `Author` as a sum type.
+Our API tells us the author of the blog post is a `String` and whether we follow them is a `Boolean`. This admits more cases than are actually possible -- you can't follow yourself, for example -- so we are more precise and model an `Author` as a sum type.
```

Comment:
I can see the original phrasing making more sense if "tells" is replaced with "sends".

Reply:
Yes, "sends" is a better word here.